test/test_nn.py

```diff
@@ -1977,6 +1977,19 @@ def test_orthogonal(self):
                         self.assertEqual(torch.mm(flattened_tensor, flattened_tensor.t()),
                                          torch.eye(rows) * gain ** 2, prec=1e-6)
 
+    def test_PReLU(self):
+        p = nn.PReLU(3, 0.3)
+        input = Variable(torch.randn(2, 3, 4), requires_grad=True)
```

Comment:
you also prob want to test for the case you fixed in this PR, which is a 5D Tensor